app/models/form.rb

```diff
@@ -79,7 +79,7 @@ def next_incomplete_section_redirect_path(subsection, case_log)
     when :in_progress
       "#{next_subsection.id}/check_answers".dasherize
     when :not_started
-      first_question_in_subsection = next_subsection.pages.first.id
+      first_question_in_subsection = next_subsection.pages.find { |page| page.routed_to?(case_log, nil) }.id
```

Comment:
is routed_to? guaranteed to return a result?

Reply:
Not strictly but if you had a section where no pages were routed to you've probably got an issue with your form design already (and a hard crash might be good to alert us to that fact)